client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -326,17 +330,12 @@ const Checkout = React.createClass( {
 } );
 
 module.exports = connect(
-	state => {
-		const selectedSiteId = getSelectedSiteId( state );
-
-		return {
-			cards: getStoredCards( state ),
-			isDomainOnly: getSiteOption( state, selectedSiteId, 'is_domain_only' ),
```

Comment:
@yurynix We definitely need to remove this, but I think we need to replace it with a strategy for determining if we're in a domain-only checkout.